src/components/custom-field-input-text/custom-field-input-text.css

```diff
@@ -40,10 +43,13 @@
   color: var(--mds-grey-54);
 }
 
-.input-icon {
-  composes: icon-base from '../icon/icon.css'; /* stylelint-disable-line css-modules/composed-class-names */
-  pointer-events: none;
+.icon-container {
   position: absolute;
   right: 6px;
   top: 6px;
 }
+
+.input-icon {
+  composes: icon-base from '../icon/icon.css'; /* stylelint-disable-line css-modules/composed-class-names */
+  pointer-events: none;
```

Comment:
This turns off click events, right? Why do we want those to not fire?

Reply:
I am not sure, this was just moved.